packages/devtools_app/lib/src/ui/utils.dart

```diff
@@ -191,6 +191,7 @@ void scrollToPosition(
     scrollController.offset + scrollController.position.extentInside,
   );
 
+  // Scroll to position if not in view already
```

Comment:
if you add dart doc above (see earlier comment), you can remove this as it will become redundant.